### ISSUE TITLE
How to use the fullsize of the terminal if output the source part to another terminal?

### ISSUE THREAD

**Issue by c02y** [cyrus-and/gdb-dashboard#51]
For example, I open two panes side-by-side using tmux/screen, and I use `dashboard source -output source.c /dev/pts/10`  to the right pane which is `/dev/pts/10` to view the source code part, but only 5 (which is default) lines of code is shown in `/dev/pts/10`, I can change the default value, but it is fixed anyway. 

How can I make the source part use the rest (in this case, fullsize) of the terminal if I output it to another terminal?

**Comment by cyrus-and** [comment]
This is not possible at the moment, you'll have to manually specify the context.
Planned feature though...

**Comment by phcerdan** [comment]
+1 on this!

**Comment by hortinstein** [comment]
FYSA if anyone is looking to manually change this:

```
 def attributes(self):
        return {
            'context': {
                'doc': 'Number of context lines.',
                'default': 5, #EDIT THIS
                'type': int,
                'check': check_ge_zero
            }
}
```

**Comment by cyrus-and** [comment]
@hortinstein you can easily [customize][1] that, there's no need to change the default value.

[1]: https://github.com/cyrus-and/gdb-dashboard#stylable-attributes

**Comment by cyrus-and** [comment]
Implemented in c824ff91ac150c29152e78f569ffada6e4519cec! Just pass 0 to the height of the Source or Assembly modules to make them use the whole available space. Of course it only does make sense when a module is the only one bound to a given terminal.

```
dashboard source -style height 0
dashboard assembly -style height 0
```
